demisto_sdk/commands/lint/linter.py

```diff
@@ -637,7 +641,9 @@ def _run_xsoar_linter(self, py_num: str, lint_files: List[Path]) -> Tuple[int, s
             )
             stdout, stderr, exit_code = run_command_os(
                 command=build_xsoar_linter_command(
-                    lint_files, self._facts.get("support_level", "base")  # type: ignore
+                    lint_files,  # type: ignore
+                    self._facts.get("support_level", "base"),
```

Comment:
please explicitly pass the values, e.g. `files=line_files, support_level=self._facts.get(...)`